sklearn/compose/_column_transformer.py

```diff
@@ -135,6 +139,10 @@ class ColumnTransformer(_BaseComposition, TransformerMixin):
     dropped from the resulting transformed feature matrix, unless specified
     in the `passthrough` keyword. Those columns specified with `passthrough`
     are added at the right to the output of the transformers.
+    Although this, as well as the option to specify column names,
```

Comment:
Does this comment pertain to passthrough?

Reply:
I intended this note to be a general (not passthrough-specific) heads-up to not draw incorrect conclusions from the ability to reference columns by name. If you are asking whether column reordering also causes problems with the remainder='passthrough' option, then this to my understanding is the case, too. Does this answer your question? If the text I added is superfluous or unclear, I can remove it.

Reply:
Let's leave this text out for now, if that's alright. I think it could be clearer, but would rather fix the issue at hand than worry about it.